tests/jest.config.js

```diff
@@ -1,9 +1,16 @@
-/** @type {import('jest').Config} */
+/** @type {import('ts-jest').JestConfigWithTsJest} */
 module.exports = {
     testEnvironment: 'jsdom',
     transformIgnorePatterns: [],
     snapshotSerializers: ['jest-serializer-html'],
     transform: {
-        '^.+\\.(j|t)s?$': ['esbuild-jest', {tsconfig: './tsconfig.json'}],
+        '^.+\\.(j|t)s?$': [
```

Comment:
FYI
I encountered issues with typings in the tests. I implemented it by analogy with https://github.com/diplodoc-platform/tabs-extension/commit/317c0dad275bfa1e8a62e4beda672124ce29e454.